lib/middleware.js

```diff
@@ -79,7 +79,9 @@ module.exports = function wrapper(context) {
           contentType += '; charset=UTF-8';
         }
 
-        res.setHeader('Content-Type', contentType);
+        if (!res.getHeader('Content-Type')) {
+          res.setHeader('Content-Type', contentType);
+        }
         res.setHeader('Content-Length', content.length);
```

Comment:
What do you think should we add check here too? I think in theory it is also possible to set own `Content-Length`, don't known real use case, but still think what we should add check here too

Reply:
This was my initial thought too. But I think matters are more complicated... It looks like ExpressJS itself sets the `Content-Length` 

https://github.com/expressjs/express/blob/3d10279826f59bf68e28995ce423f7bc4d2f11cf/lib/response.js#L194

In fact line 85 seems to be not functional at all.

Reply:
@dmohns good catch, can you create issue? Will be great if you help to use investigate this, let's merge this as is

Reply:
👍 Will do.

Reply:
hello，this commit make error，"TypeError: res.getHeader is not a function"，my node version is v10.15.2，how can i solve this problem

Reply:
> 👍 Will do.

hello，this commit make error，"TypeError: res.getHeader is not a function"，my node version is v10.15.2，how can i solve this problem